src/ScalarPotentials/ImpurityScale.jl

```diff
@@ -46,4 +46,20 @@ function ImpurityScale(nt::NamedTuple)
     N = get_number_of_dimensions(grid)
     ImpurityScale{T, N, S, typeof(grid.axes)}(nt.values, grid)
 end
-Base.convert(T::Type{ImpurityScale}, x::NamedTuple) = T(x)
\ No newline at end of file
+Base.convert(T::Type{ImpurityScale}, x::NamedTuple) = T(x)
+
+
+"""
+    get_ticks_at_positions_of_edge_of_depleted_volumes(field::ImpurityScale{T, 3})
+
+The impurity scale field is analyzed in order to find and return ticks where the gradient is strong,
```

Comment:
Can you specify this a bit more? What does `analyze` mean and what is `strong`?
You use the value 0.5 later on in this function, maybe add that here into the definition of `strong`..